Makefile

```diff
@@ -1,27 +1,68 @@
 
+ifndef VERBOSE
+.SILENT:
+endif
```

Comment:
This will block the stdout from make, so the output will be only from the script (unless we add the `VERBOSE` flag)